src/storage/scripting.cc

```diff
@@ -109,12 +110,10 @@ void loadFuncs(lua_State *lua, bool read_only) {
   lua_pushcfunction(lua, redisStatusReplyCommand);
   lua_settable(lua, -3);
 
-  if (read_only) {
-    /* redis.read_only */
-    lua_pushstring(lua, "read_only");
-    lua_pushboolean(lua, 1);
-    lua_settable(lua, -3);
-  }
+  /* redis.read_only */
+  lua_pushstring(lua, "read_only");
+  lua_pushboolean(lua, read_only);
+  lua_settable(lua, -3);
```

Comment:
It is not intuitionistic if the boolean value only exists when read_only is true